File: Pattern/pattern6/pattern.go
package main

import "fmt"

func pattern(n int) {
	for i := 0; i < n; i++ {
		for j := 1; j <= n-i; j++ {
			fmt.Print(j)
		}
		fmt.Println("")
	}
}

func main() {
	pattern(5)
}

File: Pattern/pattern6/Pattern.java
package Pattern.pattern6;

public class Pattern {
    public static void main(String[] args) {
        pattern(8);
    }

    public static void pattern(Integer n) {
        for (int i = 0; i <= n; i++) {
            for (int j = 1; j <= n - i; j++) {
                System.out.print(j);
            }
            System.out.println("");
        }
    }
}
